src/pages/policy.page.js

```diff
@@ -499,22 +503,27 @@ goToCurrentAdaptationPolicy () {
 
   async getTotalNumberOfRecordsOfPolicy() {
     const numberOfRecordsOfPolicy = await I.grabNumberOfVisibleElements(this.table.tableRows);
+    I.say('Rows count is '+numberOfRecordsOfPolicy)
     return numberOfRecordsOfPolicy
   },
 
   selectCountOfPolicies(itemCount) {
     const element = this.options.countOfPolicies;
-    I.selectOption(element, itemCount);
+    I.clickElement(element)
+    let countSelection = `//li[contains(.,'`+ itemCount +`')]`
+    I.clickElement(countSelection);
+    I.waitForElement(element, 60)
+
   },
 
-  assertPoliciesItemShownCount(itemCount, availableRecords) {
-    availableRecords.then((records) => {
-      if (records > itemCount) {
-        I.seeNumberOfElements(this.table.tableRows, itemCount)
+  async isRecordCountAccurate(recordsCount) {
+    const displayedCount = await I.grabTextFrom(this.options.countView);
+    let itemsCount = await this.getTotalNumberOfRecordsOfPolicy();
+      if (itemsCount === recordsCount && displayedCount.includes('1-'+itemsCount)) {
+        I.say('The selected number of record is correctly displayed as: '+itemsCount)
       } else {
-        I.seeNumberOfElements(this.table.tableRows, records)
+        assert.fail('The number of records displayed '+itemsCount+ ' is not as expected')
```

Comment:
can use a template literal here

```suggestion
        assert.fail(`The number of records displayed ${itemsCount} is not as expected`)
```